lib/stimulus_reflex/cable_readyable.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+require "active_support/concern"
+
+module StimulusReflex
+  module CableReadyable
+    extend ActiveSupport::Concern
+
+    prepended do
+      attr_reader :cable_ready
+    end
+
+    def initialize(*args, **kwargs)
+      super(*args, **kwargs)
+
+      if is_a? CableReady::Broadcaster
+        message = <<~MSG
+
+          #{self.class.name} includes CableReady::Broadcaster, and you need to remove it.
+          Reflexes have their own CableReady interface. You can just assume that it's present.
+          See https://docs.stimulusreflex.com/rtfm/cableready#using-cableready-inside-a-reflex-action for more details.
+
+        MSG
+        raise TypeError.new(message.strip)
+      end
+
+      @cable_ready = StimulusReflex::CableReadyChannels.new(stream_name, reflex_id)
```

Comment:
We should probably add methods for `stream_name` and `reflex_id` to indicate that anything prepending this module should implement these methods?

```ruby
def stream_name
  super # prepending modules are required to implement this
end

def reflex_id
  super # prepending modules are required to implement this
end
```

Reply:
I concur, it just seems to me that this subset of attributes is pretty arbitrary. Maybe we should just pass the reflex instance to CableReadyChannels and refactor there?

Reply:
Agree. That's a much better idea.

Reply:
Done!

Reply:
In the future, maybe pattern matching could be of help here. I think it will really pay off wrt Morph Types